src/separations.cc

```diff
@@ -0,0 +1,283 @@
+#include "separations.h"
+
+using cyclus::Material;
+using cyclus::Composition;
+using cyclus::toolkit::ResBuf;
+using cyclus::toolkit::MatVec;
+using cyclus::KeyError;
+using cyclus::ValueError;
+using cyclus::Request;
+using cyclus::CompMap;
+
+namespace cycamore {
+
+Separations::Separations(cyclus::Context* ctx) : cyclus::Facility(ctx) {
+  cyclus::Warn<cyclus::EXPERIMENTAL_WARNING>(
+      "the Separations archetype "
+      "is experimental");
+}
+
+cyclus::Inventories Separations::SnapshotInv() {
```

Comment:
I'm just curious, is there a reason at least part of these Inv-related functions can not be auto-gen'd with `pragma cyclus impl`?